dvc/path/gdrive.py

```diff
@@ -0,0 +1,10 @@
+from dvc.scheme import Schemes
+from .base import PathBASE
+
+
+class PathGDrive(PathBASE):
+    scheme = Schemes.GDRIVE
+
+    def __init__(self, root, url=None, path=None):
+        super(PathGDrive, self).__init__(url, path)
+        self.root = root
```

Comment:
I don't think this is used.

Reply:
Could you give some details? As I understand it is used in `from_info`/`to_info` `.root` references.

Reply:
Aaah, right. Didn't notice that. :+1: